.travis.yml

```diff
@@ -14,6 +14,8 @@ after_success:
   - pip install codecov
   - codecov
 cache: pip
+env:
+  TOXENV: py
```

Comment:
this is really, cool, but look like a magic for me ;)

Reply:
It just matches any `py*` envs. Well, it could be omitted because it's default behavior.
But I normally don't even list envs. `envlist = python` is enough to make everything work without extra configuration.